src/main/java/com/ibm/watson/developer_cloud/text_to_speech/v1/model/CustomVoiceModel.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.ibm.watson.developer_cloud.text_to_speech.v1.model;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * A customized voice model, that allows users to specify custom pronunciations
+ * for Waton's Text to Speech API.
+ * 
+ * @see <a href=
+ *      "http://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/doc/text-to-speech/custom-intro.shtml">
+ *      Customization</a>
+ */
+public class CustomVoiceModel extends GenericModel {
+
+  @SerializedName("customization_id")
+  private String id;
+
+  private String name;
+
+  private String description;
+
+  private String language;
+
+  private String owner;
+
+  private Timestamp created;
```

Comment:
What if we use `Long` for the timestamps. I don't like the idea of using a `java.sql` class but I think that having `Date`s will complicate things